### PR TITLE
Only allow people to star after 24-hour guild presence

### PR DESCRIPTION
**Pull request by robflop**
Seeing as we've had two userbots starring every message today, I believe this is appropriate.

### CODE REVIEW

**Comment by 1Computer1** [comment]
Considering the (most likely) spamming, I'm not sure if replying would do any good.

**Comment by 1Computer1** [comment]
Also, add this for reactions too!

File: commands/star/star.js [hunk 1]
```diff
@@ -34,6 +35,10 @@ module.exports = class StarCommand extends Command {
 			return msg.delete().catch(err => null); // eslint-disable-line
 		}
 
+		if (moment().diff(message.member.joinedAt, 'hours') < 24) {
+			return msg.reply('sorry, you can only star messages after being on the guild for over a day!');
```
Comment (kurisubrooks) [inline commands/star/star.js]: Capitalise the S in Sorry
Change "on the guild" to "in the guild"

Comment (1Computer1) [inline commands/star/star.js]: It's a reply tho, it would be like, "@person, sorry..."

Comment (kurisubrooks) [inline commands/star/star.js]: Ah, fair point.

Comment (alucard0134) [inline commands/star/star.js]: why is there an exclamation point, nothing about this is exciting

Comment (robflop) [inline commands/star/star.js]: The other replies also use exclamation points, that's why

**Comment by alucard0134** [comment]
i love your squashing daddy~~

**Pull request merged**